pkg/core/localize/locales/en/cmd/kafka.en.toml

```diff
@@ -528,6 +532,34 @@ one = 'Format in which to display the Kafka topic (choose from: "json", "yml", "
 [kafka.topic.list.flag.output.description]
 one = 'Format in which to display the Kafka topic (choose from: "json", "yml", "yaml")'
 
+[kafka.topic.produce.flag.key.description]
+one = 'The key associated with the value produced. Empty if not set'
+
+[kafka.topic.produce.flag.key.file]
+one = 'Path to file containing value'
+
+[kafka.topic.produce.flag.partition.description]
+one = 'The partition receiving the message'
+
+[kafka.topic.produce.info.produceSuccess]
+description = 'Produce success message'
+one = 'Record successfully produced to "{{.Topic}}"'
+
+[kafka.topic.consume.flag.partition.description]
+one = 'The partition to consume from'
+
+[kafka.topic.consume.flag.timestamp.description]
+one = 'Timestamp to start consuming from'
+
+[kafka.topic.consume.flag.wait.description]
+one = 'Continuously waiting for records to consume'
+
+[kafka.topic.consume.flag.offset.description]
+one = 'Retrieve messages with an offset equal to or greater than this'
+
+[kafka.topic.consume.flag.limit.description]
+one = 'Max records to consume from topic'
```

Comment:
limit is applied to single request always. Then with --wait we need to document that server is being pulled with arguments specified by user (including limit)

Reply:
Even if we were to clearly document this, it still seems kinda weird no? Limit to me always implies only ever give me back this amount of messages no matter what! As a user, I don't really think about individual requests.

Reply:
What is the suggestion here? Ignore limit when --wait is supplied?

Reply:
No don't ignore it all together just accumulate all messages received up until that limit without any consideration of each request.

For example, I have 5 messages on my topic but I set my limit to 6. I should consume all 5 on this topic and then when another message is produced consume that giving a total of 6. Any other message produced is not consumed and consuming is stopped.

Reply:
Would that represent some valid end user use case?

Reply:
I think so, don't see a big difference between this and a normal limit flag without waiting. 

Also using a limit for each request as you suggest just ignores the limit, if my limit is 1 but there are more than that on the topic then it will just keep consuming but just as separate requests. 

Limit in this scenario just means keep consuming just slower because we are doing more requests.

Reply:
Let's close comments that were addressed.